infoblox_client/feature.py

```diff
@@ -89,18 +92,18 @@ def is_version_supported(self, req_ver):
             elif b is None:
                 return True
             elif not a == b:
-                return (a > b)
+                return a > b
         return True
 
     @staticmethod
     def _get_wapi_version_parts(version):
         parts = version.split('.')
-        if (not parts or len(parts) > 3 or len(parts) < 2):
+        if not parts or len(parts) > 3 or len(parts) < 2:
             raise ValueError("Invalid argument was passed")
         for p in parts:
             if not len(p) or not p.isdigit():
                 raise ValueError("Invalid argument was passed")
         parts = [int(x) for x in parts]
         if len(parts) == 2:
-            parts.append(None)
+            parts.append(0)
```

Comment:
this broke a UT

Reply:
True, I fixed the test as well and pushed the changes.